curvefs/src/mds/fs_storage.cpp

```diff
@@ -570,5 +620,35 @@ FSStatusCode PersisKVStorage::DeleteFsUsage(const std::string& fsName) {
     return FSStatusCode::OK;
 }
 
+FSStatusCode PersisKVStorage::Tso(uint32_t fsId, uint64_t* ts,
+                                  uint64_t* timestamp) {
+    WriteLockGuard lock(tsLock_);
```

Comment:
This lock also causes requests from different file systems to be processed serially.

Reply:
Fixed, change to one tsIdGenerator in cluster.